browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -2518,6 +2518,18 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_MARKET" desc="Brave crypto wallet section Market">
       Market
     </message>
+    <message name="IDS_INFORMATION" desc="Brave crypto wallet information label">
+      INFORMATION
```

Comment:
`INFORMATION` is title case on desktop. Either desktop or mobile (Android+iOS) needs to be updated though usually we use title case.

Reply:
![Simulator_Screenshot_-_iPhone_14_Pro_-_2023-06-12_at_11_15_10](https://github.com/brave/brave-core/assets/3308503/ef917c8a-41a2-4db2-b6ef-c8ac82297af7)

I agree it is pretty strange. 
I copied it from iOS though.

The strings were also approved though.
Do we have a GitHub issue for this? So we can raise attention and change it on all platforms where it makes sense.

Reply:
yeah, aware that it's from iOS. @rmcfadden3 Could you please help with this? TIA! (though Slack could be faster to cofirm)

Reply:
Hi @Pavneet-Sing — you're saying basically this should be `Information` instead of `INFORMATION`, is that right? If yes, that makes sense to me. But it's not like a big deal if you can't get the update in here (or globally).